### PR TITLE
Improve sidebar rendering performance [SCI-2478]

### PR DESCRIPTION
**Pull request by okriuchykhin**
https://biosistemika.atlassian.net/browse/SCI-2478

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -75,7 +75,7 @@
   config.assets.raise_runtime_errors = true
 
   # Only log info and higher on development
-  config.log_level = :info
+  config.log_level = :debug
```
Comment (Ducz0r) [inline config/environments/development.rb]: I also use this locally a lot, but I'm not sure if this is needed to be changed?

File: app/views/shared/sidebar/_projects.html.erb [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
-            </span>
-            <%= render partial: 'shared/sidebar/experiments',
-                      locals: { project: project } %>
-          </li>
-        <% end %>
+  <% if @projects_tree.present? then %>
+    <% @projects_tree.each do |project| %>
+      <% if (project_page? ||
+             sample_types_page_project? ||
+             sample_groups_page_project?) && project == @project %>
```
Comment (Ducz0r) [inline app/views/shared/sidebar/_projects.html.erb]: Probably we can get rid of these 2 checks: `sample_types_page_project?`, `sample_groups_page_project?`, right?

File: config/environments/development.rb (hunk 1 above)
Comment (okriuchykhin) [reply config/environments/development.rb]: I don't know, it's default value in Rails 5

File: app/views/shared/sidebar/_projects.html.erb (hunk 2 above)
Comment (okriuchykhin) [reply app/views/shared/sidebar/_projects.html.erb]: Done

**Comment by Ducz0r** [comment]
Great, now it works perfectly :+1:!

**Pull request merged**